frontend/src/components/common/info/messages/EmptySessionsTabMessage.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { type ReactElement } from "react";
+import { Center, useStyleConfig } from "@chakra-ui/react";
+
+import DistributeAssessmentsIllustration from "../../../../assets/illustrations/distribute-assessments.svg";
+import type { TestSessionStatus } from "../../../../types/TestSessionTypes";
+import MessageContainer from "../MessageContainer";
+
+const BODY_PARAGRAPHS_BY_STATUS: Record<TestSessionStatus, string[]> = {
```

Comment:
tbh i'm also fine with just removing the body paragraph, i think the subtitle provides enough info and it looks a bit text heavy rn

Reply:
I think it might be worth clarifying; also the empty state component marked the `paragraphs` field as required so I feel like there could be design intent to have explanation text